### PR TITLE
Update README volumetric and main 

### PR DESCRIPTION
**Pull request by Roxana-P**
This PR gives an overview about the content in the different samles in the main README. 
Furthermore it adds the file from PR 4 in the `volumes/tet`  folder.

### CODE REVIEW

**Review by clemens-fricke — DISMISSED** [review]
Thank you for the documentation addition.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+
+
+## Content
+*Faces*
+- quad
+- tri
+
+*Volumes*
+- hex
+- quad
```
Comment (j042) [inline README.md]:
```suggestion
- tet
```

**Review by j042 — APPROVED** [review]
thanks!

**Comment by j042** [comment]
need rebase

**Pull request merged**